### ISSUE TITLE
unnecessary warning from webpack from normalize()

### ISSUE THREAD

**Issue by mutsys** [typestyle/csstips#2]
When using csstips with webpack, webpack will generate an annoying (albeit harmless) warning message:

```
./~/csstips/lib/normalize.js
Cannot find SourceMap 'normalize.min.css.map': Error: Can't resolve './normalize.min.css.map' in '/the/absolute/path/to/node_modules/csstips/lib'
```

It appears that even though normalize.js strips out the css source map comment before passing it to `cssRaw`, this does not prevent webpack from seeing it and trying to resolve the value of sourceMappingURL, resulting in the above warning.

Sure, this doesn't have any functional impact, but since normalize.js is created manually from normalize.css by copying/pasting with the source map comment intact, and then adding code to strip that comment out, how about you just leave out the comment to begin with?

I would be happy to send you a PR to clean it up.

**Comment by basarat** [comment]
> I would be happy to send you a PR to clean it up.

Would appreciate that :rose: :heart:

**Comment by mutsys** [comment]
My pleasure. The typestyle/csstips/csx suite of tools are an awesome bit of work and I am happy to be able contribute a little something back. PR sent.

**Comment by basarat** [comment]
released `0.2.0` :rose:
